src/main/app/claims/models/claim.ts

```diff
@@ -117,6 +121,20 @@ export class Claim {
     }
   }
 
+  get stateHistory (): State[] {
```

Comment:
Why is the `State` interface needed here? Cannot we just return a `ClaimStatus[]`?

Reply:
We could, I created `State` initially as it had the date of the event as well. Then because we do not have dates for offers (we should by the way) I removed that piece without realising that `State` is now only wrapper for `ClaimStatus`.

Reply:
Would be nice to remove it in that case. I know not having dates there is awkward but there never was a requirement for them specified in any of the offer stories...